### ISSUE TITLE
will open source the fine-tuned BlEU-RT model?

### ISSUE THREAD

**Issue by choucaicai** [HKUST-KnowComp/ActPlan-1K#1]
Thanks for your work！I noticed that the README.md doesn't mention whether the fine-tuned BLEU-RT model will be released. This model could be valuable for the research community and help reproduce the results in the paper. Are there any plans to open-source the model weights and training scripts?

**Comment by WeihangGuo** [comment]
Yes, it would be really helpful for others to use the benchmark.
